lib/pbench/cli/server/__init__.py

```diff
@@ -51,29 +75,32 @@ def message(self, message: str):
 class Verify:
     """Encapsulate -v status messages."""
 
-    def __init__(self, verify: bool):
+    def __init__(self, verify: Union[bool, int]):
         """Initialize the object.
 
         Args:
             verify: True to write status messages.
         """
-        self.verify = verify
+        if isinstance(verify, int):
+            self.verify = verify
+        else:
+            self.verify = 1 if verify else 0
 
     def __bool__(self) -> bool:
         """Report whether verification is enabled.
 
         Returns:
             True if verification is enabled.
         """
-        return self.verify
+        return bool(self.verify)
 
-    def status(self, message: str):
+    def status(self, message: str, level: int = 1):
```

Comment:
I think the value used at line 97 is the "default verification level".  I think the value set at line 87 is also the "default verification level".  If they are indeed the same "thing", then having a defined symbol for them would be appropriate.

You're saying that they are different and the fact that the values are the same is just a coincidence...I'm wondering if there is something that we could do to make it more obvious that the two values have nothing to do with each other...because, as you say, it's a small class and the references are not far away from each other.